script/src/main/java/io/kestra/plugin/scripts/runner/docker/Docker.java

```diff
@@ -313,6 +314,14 @@ How to handle local files (input files, output files, namespace files, ...).
     )
     private Duration killGracePeriod = Duration.ZERO;
 
+    @Builder.Default
+    @Schema(
+        title = "Whether to resume an existing matching container on restart.",
+        description = "If enabled, the runner will search for an existing container labeled with the current execution/task identifiers and reattach to it instead of creating a new container."
+    )
+    @PluginProperty
+    private Property<Boolean> resume = Property.ofValue(false);
```

Comment:
Resume is enabled by default on all task runners that support it, is there some specific consideration so that you disable it by default?